README.md

```diff
@@ -23,8 +23,8 @@ mkvirtualenv forest-service-prototype
 pip install -r requirements.txt
 createdb forest-service-prototype
 cd forestserviceprototype
-./manage.py migrate
-./manage.py runserver
+forestserviceprototype/manage.py migrate
```

Comment:
You're right; I must've mixed up lines 21 and 25 when I was first setting up my local environment.